website/docs/d/scc_posture_scan_summaries.html.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_scan_summaries"
+description: |-
+  Get information about scan_summaries
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_scan_summaries
+
+Review information of Security and Compliance Center scan summaries.
```

Comment:
please do provide the hyperlink using []() syntax as shown in above, and should not provide stage docs link for the registry docs.